lab5.cpp

```diff
@@ -0,0 +1,156 @@
+﻿
+
+/*Створити клас Fish(name, age, species, size, prefferedFood, isAggressive, neededSpace(cubic meters)).Створити клас
+Aquarium,який мiстить різних рибок, та має поля totalVolume(cubic meters) і freeSpace(cubic meters).Додати методи
+для виведення 3 - ьох найбільших риб у акваріумі.У методі main створіть декілька акваріумів, враховуючи вимоги
+рибок(агресивних можна додавати лише до агресивних, вільне місце в акваріумі повинне завжди залишатися >= 0) 
+та продемонструйте роботу класів.*/
+
+
+#include <iostream>
+#include <string>
+#include <vector>
+using namespace std;
+
+class Fish
+{
+protected:
+    string name, spicies, preffered_food;
+    int age, needed_space, size;
+    bool is_aggressive;
+    
+public:
+    string getname()
```

Comment:
please use getName or GetName or get_name, not getname() style of naming functions for this getter and all of next getters